en/on-prem-prerequisites-and-installation.md

```diff
@@ -0,0 +1,135 @@
+---
+title: App Builder On-premise Prerequisites and Installation
+_description: Learn how to start using App Builder On-premises version and what are the prerequisites to install and run it.
+_keywords: App builder, On-premise, Infragistics
+---
+
+# App Builder On-premise Prerequisites and Installation
+
+## Prerequisites
+
+This section lists the prerequisites to install the On-Prem version of AppBuilder and is intended for System Administrators who configure operational parameters that maintains and supports Linux/Mac OS/Windows.
```

Comment:
This section lists the prerequisites to install the On-Prem version of App Builder and is intended for System Administrators who configure operational parameters that maintain and support Linux/Mac OS/Windows.